test/integration.test.ts

```diff
@@ -532,3 +460,104 @@ describe('integration tests', () => {
     expect(queriedPlanes.records).toHaveLength(PAGINATION_DEFAULT_SIZE);
   });
 });
+
+describe('record creation', () => {
+  test('create single team without id', async () => {
+    const team = await client.db.teams.create({ name: 'Team ships' });
+
+    expect(team.id).toBeDefined();
+    expect(team.name).toBe('Team ships');
+  });
+
+  test('create multiple teams without ids', async () => {
+    const teams = await client.db.teams.createMany([{ name: 'Team cars' }, { name: 'Team planes' }]);
+
+    expect(teams).toHaveLength(2);
+    expect(teams[0].id).toBeDefined();
+    expect(teams[0].name).toBe('Team cars');
+    expect(teams[1].id).toBeDefined();
+    expect(teams[1].name).toBe('Team planes');
+  });
+
+  test('create user with id', async () => {
+    const user = await client.db.users.create('a-unique-record-john-4', {
+      full_name: 'John Doe 4',
+      email: 'john4@doe.com'
+    });
+
+    const apiUser = await client.db.users.filter({ id: user.id }).getOne();
+    if (!apiUser) throw new Error('No user found');
+
+    expect(user.id).toBe('a-unique-record-john-4');
+    expect(user.full_name).toBe('John Doe 4');
+
+    expect(user.id).toBe(apiUser.id);
+    expect(user.full_name).toBe(apiUser.full_name);
+    expect(user.email).toBe(apiUser.email);
+
+    expect(
+      client.db.users.create('a-unique-record-john-4', {
+        full_name: 'John Doe 5',
+        email: 'john5@doe.com'
+      })
+    ).rejects.toMatchInlineSnapshot(
+      {
+        message: expect.any(String),
+        status: 422
+      } as any,
+      `
+      Object {
+        "message": Any<String>,
+        "status": 422,
+      }
+    `
+    );
+  });
+
+  test('create user with inlined id', async () => {
+    const user = await client.db.users.create({
+      id: 'a-unique-record-john-5',
+      full_name: 'John Doe 5',
+      email: 'john5@doe.com'
+    });
+
+    const apiUser = await client.db.users.filter({ id: user.id }).getOne();
+    if (!apiUser) throw new Error('No user found');
+
+    expect(user.id).toBe('a-unique-record-john-5');
+    expect(user.full_name).toBe('John Doe 5');
+
+    expect(user.id).toBe(apiUser.id);
+    expect(user.full_name).toBe(apiUser.full_name);
+    expect(user.email).toBe(apiUser.email);
+  });
+
+  test('create user with empty id is not allowed', async () => {
+    expect(
+      client.db.users.create('', {
+        full_name: 'John Doe 3',
+        email: 'john3@doe.com'
+      })
+    ).rejects.toMatchInlineSnapshot(`[Error: The id can't be empty]`);
+  });
+
+  test('create user with empty inline id is not allowed', async () => {
+    expect(
+      client.db.users.create({
+        id: '',
+        full_name: 'John Doe 3',
+        email: 'john3@doe.com'
+      })
+    ).rejects.toMatchInlineSnapshot(`[Error: The id can't be empty]`);
```

Comment:
```suggestion
    ).rejects.toMatchInlineSnapshot(`[Error: The id can't be empty, null or undefined]`);
```

Reply:
If null or undefined the error "[Error: Invalid arguments for create method]". We already have a test for it: https://github.com/xataio/client-ts/pull/136/files#diff-8fdaf3b73054c861d4b1afc72e76086d85c318c75bde88a452ef344c24b7d8b0R554-R563